requirements.md

```diff
@@ -73,16 +90,22 @@ authorization it can be validated against relying-party stored instrument displa
 ## SPC Credentials
 
 * When more than one SPC Credential matches input data, the browser must choose the authenticator in the order of the input SPC Credential Identifiers.
+
+Q: Does it mean that if more than one SPC Cred IDs are valid on the given device/browser then the browser picks first on the list and does not present selection?
+
 * When no SPC Credential matches input data, the protocol should terminate without any user experience to allow for seamless fallback behaviors.
 * If the protocol supports more than one instrument per authenticator (e.g., within the same SPC Credential), then each instrument must be uniquely addressable and have unique display information.
 
+Q: I need clarification here, please. If two instruments can be bound to the single SPC Credential then how SPC is going to be invoked? By SPC Credenial ID of Instrument ID?
+
```

Comment:
I can imagine a world where we structure the data so that there are N instruments per authenticator credential. In this case, the SPC Credential ID would reference a specific instrument associated with the SPC Credential. 

In short: it seems like we should discuss the use case of N instruments per authenticator credential and see if we want to support it. If we do, then I think we can still talk about "SPC Credential ID" and the browser will figure it out.

Reply:
If I understood correctly it'd be contradictory with the previous point (https://github.com/w3c/secure-payment-confirmation/pull/71#discussion_r632531886)

Reply:
Not resolving this yet; see https://lists.w3.org/Archives/Public/public-payments-wg/2021May/0016.html

Reply:
Hi @tblachowicz, there is now a requirement that each instrument/auth credential pair be addressable. I am going to close this part of the conversation.